### PR TITLE
Small documentation fixes in README.md

### PR DESCRIPTION
**Pull request by thesharp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ get all the news about botogram in its [Telegram channel][channel].
 > Please note botogram currently doesn't support some of the upstream API
 > features. All of them will be implemented in botogram 1.0
 
-**Supported Python versions**: 3.4, 3.5
```
Comment (pietroalbini) [inline README.md]: That was intended.

In Markdown, [you need two trailing whitespaces](http://daringfireball.net/projects/markdown/syntax#p) after a line in order to put an HTML `<br>`. Apparently GitHub doesn't need those, but I'd like to have this readme as platform-agnostic as possible.

**Comment by pietroalbini** [comment]
Thank you for this pull request! As I said in the comments, the first commit is useless, but thank you for spotting that wild `make`!

Can you put a more descriptive commit message in the secondo one? Like "Replace make with invoke in the README".

**Comment by thesharp** [comment]
@pietroalbini removed the whitespace-releated commit and made a different commit message for the second one.

**Comment by pietroalbini** [comment]
Thank you!

**Pull request merged**